casper/src/slowcooker/scala/coop.rchain.casper/HashSetCasperSpecification.scala

```diff
@@ -65,8 +65,13 @@ object HashSetCasperActions {
       node.addBlock(signed)
     )
 
-  def deployment(i: Int, ts: Long = System.currentTimeMillis()): Signed[DeployData] =
-    ConstructDeploy.sourceDeploy(s"new x in { x!(0) }", ts)
+  def deployment(
+      i: Int,
+      ts: Long = System.currentTimeMillis()
+  ): Signed[DeployData] =
+    ConstructDeploy.sourceDeploy(s"new x in { x!(0) }", ts, shardId = SHARD_ID)
```

Comment:
With default shardId in `ConstructDeploy.sourceDeploy` we don't these kind of changes.